codec/av1.md

```diff
@@ -106,14 +104,30 @@ The `Block` timestamp is translated from the __[PresentationTime]__ without the
+
+
+# Seeking and Sequence Header Requirements
+
+`SimpleBlocks` with the `Keyframe` flag set and `Blocks` inside `BlockGroups` that do not contain any `ReferenceBlock` elements constitute the Random Access Points (RAP) of a Matroska `Track`.
+
+Upon random access to a RAP or upon starting playback from the beginning the player/demuxer MUST prepend the `Sequence Header OBU` contained in the `CodecPrivate` to the data of the `Block` where decoding starts after having reverted any potentially applying `ContentCompression`. Afterwards playback proceeds normally including consumption of any `Sequence Header OBUs` that are found in the bitstream.
+
+Seeking to a non-RAP is undefined and discouraged.
+
+A muxer MUST make sure that when using a conformant demuxer/player the correct `Sequence Header OBU` is active both during linear access and also after random access to a RAP. In particular, if a `Sequence Header OBU` whose contents differ from the contents of the `Sequence Header OBU` in the `CodecPrivate` is active during consumption of the first `Frame Header OBU` of a RAP when performing linear access from the beginning, then said `Block` MUST contain said `Sequence Header OBU` in front of the first `Frame Header OBU` so that it is also active when performing random access to said RAP.
+
+A muxer MAY omit (strip away and discard) `Sequence Header OBUs` provided the above criteria are still fulfilled afterwards.
+
+*Notes: a) The contents of two `Sequence Header OBUs` of an AV1 bitstream compliant with the requirements for AV1 tracks in Matroska can only differ if __[decoder_model_present_for_this_op]__ is 1 for some operating point for one (hence every) `Sequence Header OBU`. In particular, if __[timing_info_present]__ is set to 0 for any (hence every) `Sequence Header OBU`, all `Sequence Header OBUs` can be omitted from the bitstream.*
 
-The first `Sequence Header OBU` of a `CVS` is stored in the `CodecPrivate` of a `Track`. So this AV1 `Track` has the same requirements as the `CVS`.
+*b) A `Sequence Header OBU` that fulfills any of these criteria can be omitted without changing compliance to the above criteria:*
 
-If the __[decoder_model_info_present_flag]__ of this `Sequence Header OBU` is set to 1 then each keyframe `Block` MUST contain a `Sequence Header OBU` before the `Frame Header OBUs`.
+*If, after potentially stripping away all `Temporal Delimiter OBUs`, there are more than one `Sequence Header OBUs` immediately after one another, then all but the last of these `Sequence Header OBUs` can be omitted.*

```

Comment:
"Potentially" means that one can check this criterion either before or after stripping the `Temporal Delimiter OBUs` away. After is of course better as this means that more superfluous `Sequence Header OBUs` will be detected as omitable, but implementations are of course free to do is as they see fit. It does not imply that stripping away `Temporal Delimiter OBUs` is optional.

Btw: One can strip even more away before applying this test; i.e. one can discard all `OBU_PADDING` `OBUs`. Notice that the syntax of an `OBU` of type `OBU_METADATA` with [metadata_type] "Unregistered user private" may depend on the `Sequence Header OBU` currently active so that simply ignoring `OBU_METADATA` `OBUs` for this test might not be right.